RLTSquare/Unit3/view/frontend/templates/customtab.phtml

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+* @var $helper
+*/
+$helper = $this->helper('RLTSquare\Unit3\Helper\Data\Data');
+?>
+
+<strong>Username:</strong>
+<?php echo $helper->getConfig('unit3/general/username'); ?><br>
+
+<strong>Password:</strong>
+<?php
+$type = $helper->getConfig('unit3/general/dropdown');
+$pass = $helper->getConfig('unit3/general/password');
+if ($type == 'Staging' || $type == 'Development') {
```

Comment:
Not good ... instead of 'Development' and 'Stagging', you could have declared constant in your options class, You should declare value keys here as constants, e.g. "const ENV_STAGING = 'staging'", so you can later use RLTSquare\UnitThree\Model\Config\Source\Select::ENV_STAGING in if statements and other comparisons you need.